docs/build/tutorials/how-to/deploy-contract.md

```diff
@@ -0,0 +1,480 @@
+
+  // Initialize the wallet.
+  const wallet = new Wallet(PRIVATE_KEY);
+
+  // Create deployer object and load the artifact of the contract you want to deploy.
+  const deployer = new Deployer(hre, wallet);
+  const artifact = await deployer.loadArtifact("SimpleStorage");
+
+  // Estimate contract deployment fee
+  const deploymentFee = await deployer.estimateDeployFee(artifact, []);
+
+  const parsedFee = ethers.formatEther(deploymentFee);
+  console.log(`The deployment is estimated to cost ${parsedFee} ETH`);
+
+  // Deploy contract
+  const simpleStorageContract = await deployer.deploy(artifact, []);
+
+  // Show the contract info.
+  const contractAddress = await simpleStorageContract.getAddress();
+  console.log(`${artifact.contractName} was deployed to ${contractAddress}`);
+}
+```
+
+To deploy contract on `zkSync Sepolia Testnet` that we already specified as default network in our **hardhat.config.ts**, run command:
+
+```bash
+yarn hardhat deploy-zksync
+```
+
+If you prefer to explicitly select network name within a command, you can run:
+
+```bash
+yarn hardhat deploy-zksync --network zkTestnet
+```
+
+After successful deployment, your console output should look something like this:
+
+```log
+Running deploy script for the SimpleStorage contract
+The deployment is estimated to cost 0.0002588676 ETH
+SimpleStorage was deployed to 0xCE5e67aF41C194aB05fCC3860ef66790A147Adf9
+Done in 8.58s.
+```
+
+Check your deployed contract on [Sepolia testnet block explorer](https://sepolia.explorer.zksync.io/) using deployed address of the contract.
+
+## Deployment using hardhat-zksync-ethers
+
+With similar steps as above, we will now use **hardhat-zksync-ethers** to deploy our contracts.
+
+**hardhat-zksync-ethers** plugin is a wrapper around [zksync-ethers](https://www.npmjs.com/package/zksync-ethers) SDK that gives additional methods to use for faster development.
+
+### Installation
+
+In the root directory of your project, execute this command:
+
+:::code-tabs
+
+@tab yarn
+
+```bash
+yarn add -D @matterlabs/hardhat-zksync-ethers hardhat zksync-ethers ethers
+```
+
+@tab npm
+
+```bash
+npm i -D @matterlabs/hardhat-zksync-ethers
+```
+
+:::
+
+Add the plugin at the top of the **hardhat.config.ts** file:
+
+```bash
+import "@matterlabs/hardhat-zksync-ethers";
+```
+
+### Configuration
+
+To deploy contracts with **hardhat-zksync-ethers**, use the similar configuration for our **hardhat.config.ts** as shown [here.](deploy-contract.md#configuration)
+The **accounts** section enables to specify wallet private keys which help us deploy contracts with automatically populated wallet within **hardhat-zksync-ethers** plugin, but it can still be manually created for other use cases.
+
+```bash
+{
+    url: 'https://sepolia.era.zksync.dev', // you should use the URL of the zkSync network RPC
+    ethNetwork: 'sepolia',
+    zksync: true,
+    accounts:["0xYOUR_ACCOUNT_PRIVATE_KEY"] // put your private key here
```

Comment:
We should avoid recommending users place their Private Key in the `hardhat.config.ts` file because that file can easily be tracked in git and accidentally included in a commit (exposing it to the public). We need to recommend users use a `.env` file that's explicitly going to be ignored with a `.gitignore` file.